release/package.json

```diff
@@ -1203,62 +1191,62 @@
 				"FSharp.externalAutocomplete": {
 					"type": "boolean",
 					"default": false,
-					"description": "Includes external (from unopen modules and namespaces) symbols in autocomplete"
+					"description": "Includes external (from unopened modules and namespaces) symbols in autocomplete"
 				},
 				"FSharp.linter": {
 					"type": "boolean",
 					"default": true,
-					"description": "Enables integration with FSharpLinter (additional warnings)"
+					"description": "Enables integration with [FSharpLint](https://fsprojects.github.io/FSharpLint/) for additional (user-defined) warnings"
```

Comment:
Is this displayed nicely in VSCode's settings panel?

Reply:
checking these as we speak

Reply:
poop

![image](https://user-images.githubusercontent.com/573979/127017760-6cf75811-fa6c-4c82-990b-6b3da074719b.png)

Reply:
there we go 

![image](https://user-images.githubusercontent.com/573979/127018940-5226d193-d33f-40be-b363-a9dc6bbb1539.png)